### DISCUSSION TITLE
What about RSS plugin compat is Insiders exclusive?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#5741] [Q&A] [answered]
The [RSS Section](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#rss) of the [Setting up a blog](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/) is marked as an "Sponsors only".

My question now is, what parts are sponsors only?

Given that the plugin isn't part of Material for MkDocs itself and the fact that all options mentioned also are part of said plugin's documentation makes me believe that this is an oversight and that RSS is actually supported?

If not, what exactly is not yet part of default Material for MkDocs that makes RSS support not possible?

To add a bit more confusion does the [Insiders page's "What's in it for me?" section](https://squidfunk.github.io/mkdocs-material/insiders/#whats-in-it-for-me) also not mention any Sponsors-only features related to RSS... At least in combination with the blog.

**Comment by Andre601** [comment]
I've tested it out and it seems to indeed be working fine, so I assume this was simply an oversight?

Tho, there is now a small thing I'm wondering about...
I want to use the RSS feed plugin to automatically post to Mastodon using the Mastofeed service. The thing is, that the generated creation date for the feed is always `%Y-%m-%d 00:00:00` which results in the feed not being considered updated, since the post date always being older than the feed creation date.

As an example: The currentl last build date for my blog according to the `feed_rss_created.xml` file is `Fri, 21 Jul 2023 18:05:50 -0000` while the `pubDate` of my latest blog post is `Fri, 21 Jul 2023 00:00:00 +0200` which obviously is older...

Would I just change `date` metadata to include time if that's possible? Or is there another, more convenient way to get the date of when the post was made?

**Reply by squidfunk** [reply]
> Would I just change date metadata to include time if that's possible? Or is there another, more convenient way to get the date of when the post was made?

The blog plugin supports `date` and `datetime` types in metadata.

https://github.com/squidfunk/mkdocs-material/blob/ef448ed9451bd45f16cceeca6270d345589555ea/src/plugins/blog/plugin.py#L183-L185

I'm not sure about the RSS plugin, though.

**Reply by Andre601** [reply]
Yeah.
I think I just remove the date meta override from the RSS plugin as that would make it use Git metadata for creation and last update dates, which hopefully would work...

**Comment by squidfunk** [comment, marked as answer]
Support for the RSS plugin will be released as part of 9.2.0. It's already part of the beta, we just forgot to remove the sponsors banner. We're currently restructuring our documentation, moving all built-in plugins into a separate section. This section of the documentation will be fixed in the same PR.

**Reply by Andre601** [reply]
Was about to ask if it was worth the effort PRing a fix while you're currently restructuring the docs anyways... Glad you edited your reply with that extra note...

**Reply by squidfunk** [reply]
You can also create a PR and we'll fix it now, no problem.

**Reply by Andre601** [reply]
Feel like it would be a waste of time for all of us. Rather wait for the restructured docs.

You could keep the mistake in the restructured docs for me to later correct. :^)

**Reply by squidfunk** [reply]
Well, okay, your choice.
